resources/views/users/show.blade.php

```diff
@@ -0,0 +1,211 @@
+                                 src="/images/users/{{ $user->image }}" data-original-title="Usuario">
+                        @endif
+                    </div>
+                    <div class="col-md-6">
+                        <strong>{{ __('User Information') }}</strong><br>
+                        <div class="table-responsive">
+                            <table class="table table-condensed table-responsive table-user-information">
+                                <tbody>
+                                <tr>
+                                    <td>
+                                        <strong>
+                                            <span class="glyphicon glyphicon-asterisk text-primary"></span>
+                                            Identificacion
+                                        </strong>
+                                    </td>
+                                    <td class="text-primary">
+                                        {{ $user->id }}
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td>
+                                        <strong>
+                                            <span class="glyphicon glyphicon-user  text-primary"></span>
+                                            {{ __('Full Name') }}
+                                        </strong>
+                                    </td>
+                                    <td class="text-primary">
+                                        {{ $user->full_name }}
+                                    </td>
+                                </tr>
+
+                                <tr>
+                                    <td>
+                                        <strong>
+                                            <span class="glyphicon glyphicon-bookmark text-primary"></span>
+                                            {{ __('Gender') }}
+                                        </strong>
+                                    </td>
+                                    <td class="text-primary">
+                                        {{ $user->gender == 1 ? __('Male') : __('Female') }}
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td>
+                                        <strong>
+                                            <span class="glyphicon glyphicon-eye-open text-primary"></span>
+                                            {{ __('User Type') }}
+                                        </strong>
+                                    </td>
+                                    <td class="text-primary">
+                                        {{ $user->is_admin == 1? __('Admin') : __('Normal User') }}
```

Comment:
is_admin is boolean type (has description in branch migration), is_admin = 1 -> admin user, is_admin = 0 -> normal user